tools/test/topology/test-multi-playback.m4

```diff
@@ -54,6 +54,16 @@ PIPELINE_PCM_ADD(sof/pipe-dai-endpoint.m4,
 	1000, 0, 0,
 	8000, 192000, 48000)
 
+# connect pipelines together
+SectionGraph."pipe-sof-second-pipe" {
+        index "2"
+
+        lines [
+		# keyword detect
+                dapm(PIPELINE_SOURCE_2, PIPELINE_CROSSOVER_1)
+        ]
+}
+
```

Comment:
The topology debug message looks correct after I moved SectionGraph here.
https://paste.googleplex.com/6407279421161472
Test Pipeline:
PCM0P->BUF1.0->CROSSOVER1.0->BUF1.1->SSP5.OUTCROSSOVER1.0->BUF2.0->SSP2.OUT

But in fact this is a bug in tplg_parser.c which will assume the routings in topology file is already sorted for pipelines.